biogeochem/EDPhysiologyMod.F90

```diff
@@ -201,7 +242,10 @@ subroutine trim_canopy( currentSite )
           trimmed = .false.
           ipft = currentCohort%pft
           call carea_allom(currentCohort%dbh,currentCohort%n,currentSite%spread,currentCohort%pft,currentCohort%c_area)
-          currentCohort%treelai = tree_lai(currentCohort%bl, currentCohort%pft, currentCohort%c_area, &
+
+          leaf_c   = currentCohort%prt%GetState(leaf_organ, all_carbon_species)
+
+          currentCohort%treelai = tree_lai(leaf_c, currentCohort%pft, currentCohort%c_area, &
```

Comment:
sorry just continuing this thread, here leaf_c in the calc of LAI for each cohort.....